repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3019,6 +3019,7 @@ public void deleteClassification(String entityGuid, String classificationName) t
                 }
 
                 if (propagateDelete) {
+                    //delete All the pending tasks of this classification vertex
```

Comment:
is this a future todo item?

Reply:
It was just outline comment, removed it now! Ignore it.